misc/install-test-dependencies.yml

```diff
@@ -0,0 +1,163 @@
+    package: name={{item}} state=present
+    with_items:
+      - make
+      - python3-ipython
+    when: ansible_distribution == 'Fedora'
+
+  - name: Install dnf-plugins-core for dnf builddep (Fedora)
+    package: name=dnf-plugins-core state=present
+    when: ansible_distribution == 'Fedora'
+
+  - name: Install build dependencies (Fedora)
+    command: "dnf -y builddep python3-blivet --nogpgcheck"
+    args:
+      warn: false
+    when: ansible_distribution == 'Fedora'
+
+  - name: Install blivet to get all dependencies (Fedora)
+    package: name=python3-blivet state=present
+    when: ansible_distribution == 'Fedora'
+
+  - name: Install test dependencies (Fedora)
+    package: name={{item}} state=present
+    with_items:
+      - dosfstools
+      - e2fsprogs
+      - xfsprogs
+      - hfsplus-tools
+      - python3-mock
+      - python3-coverage
+      - python3-pocketlint
+      - python3-bugzilla
+      - python3-pycodestyle
+      - python3-six
+      - python3-kickstart
+      - python3-pyudev
+      - python3-pyparted
+      - libselinux-python3
+      - python3-blockdev
+      - python3-bytesize
+      - python3-libvirt
+      - python3-paramiko
+    when: ansible_distribution == 'Fedora'
+
+####### CentOS 8
+  - name: Install basic build tools (CentOS 8)
+    package: name={{item}} state=present
+    with_items:
+      - make
+    when: ansible_distribution == 'CentOS' and ansible_distribution_major_version == '8'
+
+  - name: Enable EPEL repository (CentOS 8)
+    package: name=epel-release state=present
+    when: ansible_distribution == 'CentOS' and ansible_distribution_major_version == '8'
+
+  - name: Enable powertools repository (CentOS 8)
+    command: yum config-manager --set-enabled powertools
+    when: ansible_distribution == 'CentOS' and ansible_distribution_major_version == '8'
+
+  - name: Install dnf-plugins-core for dnf builddep (CentOS 8)
+    package: name=dnf-plugins-core state=present
+    when: ansible_distribution == 'CentOS' and ansible_distribution_major_version == '8'
+
+  - name: Install build dependencies (CentOS 8)
+    command: "dnf -y builddep python3-blivet --nogpgcheck"
+    args:
+      warn: false
+    when: ansible_distribution == 'CentOS' and ansible_distribution_major_version == '8'
+
+  - name: Install blivet to get all dependencies (CentOS 8)
+    package: name=python3-blivet state=present
+    when: ansible_distribution == 'CentOS' and ansible_distribution_major_version == '8'
+
+  - name: Install test dependencies (CentOS 8)
+    package: name={{item}} state=present
+    with_items:
+      - dosfstools
+      - e2fsprogs
+      - xfsprogs
+      - python3-mock
+      - python3-coverage
+      - python3-bugzilla
+      - python3-pycodestyle
+      - python3-six
+      - python3-kickstart
+      - python3-pyudev
+      - python3-pyparted
+      - libselinux-python3
+      - python3-blockdev
+      - python3-bytesize
+      - python3-libvirt
+      - python3-paramiko
+      - python3-pip
```

Comment:
You can pass `name` as a list, eg:

```
- package
    name:
       - one
       - two
       - three

```

which _should_ be substantially faster than the loop.

Reply:
Fixed. Is this a new thing? I think I remember using the `with_items` loop because the `dnf` module didn't work with lists. Anyway, it works now and it is faster.

Reply:
I'd actually suggest formatting all package lists like that, if you're touching the lines anyway.